index.js

```diff
@@ -109,6 +110,7 @@ function sortPackageJson(packageJson) {
     'eslintConfig',
     'stylelint',
     'jest',
+    'prettier',
```

Comment:
I know this is super picky, but I think we should bump this to just below `eslintConfig` - or maybe above. That way all linting tools config goes first, then test config, then dependencies.

Reply:
Makes sense, updated